planet/cli/data.py

```diff
@@ -48,6 +49,52 @@ def data(ctx, base_url):
     ctx.obj['BASE_URL'] = base_url
 
 
+def parse_item_types(ctx, param, value: str) -> List[str]:
+    """Turn a string of comma-separated names into a list of names."""
+
+    dict = {
+        "psscene": "PSScene",
+        "psorthotile": "PSOrthoTile",
+        "reorthotile": "REOrthoTile",
+        "rescene": "REScene",
+        "skysatscene": "SkySatScene",
+        "skysatcollect": "SkySatCollect",
+        "skysatvideo": "SkySatVideo",
+        "landsat8l1g": "Landsat8L1G",
+        "sentinel2l1c": "Sentinel2L1C"
+    }
+    for original, validated in dict.items():
+        update_value = value.replace(original, validated)
+
+    return [part.strip() for part in update_value.split(",")]
+
+
+def parse_filter(ctx, param, value: str) -> dict:
```

Comment:
this is new since before the PR was created, but now you can remove this logic and just use the `type.JSON` type. See how it is done in [search-create](https://github.com/planetlabs/planet-client-python/blob/f376018877daf845d3666359082475a25baf9f47/planet/cli/data.py#L272) for an example